desc/objectives/_omnigenity.py

```diff
@@ -501,6 +502,232 @@ def compute(self, params, constants=None):
         return data["f_T"]
 
 
+class Omnigenity(_Objective):
+    """Omnigenity error.
+
+    This objective assumes that the collocation point (θ=0,ζ=0) lies on the contour of
```

Comment:
is this something that needs to be enforced or checked etc? will things break if its not the case?

Reply:
OK. Is it possible to add some automated check/warning? Even if it's not exact, we could still probably catch some common pinch points
maybe something like 
```python
B = eq.compute("|B|", grid=grid)["|B|"]
node = grid.nodes[np.argmax(B)]
assert np.isclose(nodes[2], 0, atol=0.1)
```

Reply:
This is tricky because the initial guess might have a point with the highest $|B|$ that is in a "puddle" somewhere far from $(\theta=0,\zeta=0)$ but still along the proper contour line. And theoretically a bad initial guess could still converge to a good solution. 

The problematic case is if a user gives an equilibrium that is already very good omnigenity but with the B_min and B_max contours flipped, then our code will say it has high errors.